meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ullevk/lmp.cfg

```diff
@@ -10,16 +10,16 @@ CONFIG_SPL_LOAD_FIT=y
 CONFIG_SPL_RSA=y
 CONFIG_SPL_LIBCOMMON_SUPPORT=y
 CONFIG_SPL_LIBGENERIC_SUPPORT=y
-CONFIG_SPL_SHA256_SUPPORT=y
-CONFIG_SPL_CRYPTO_SUPPORT=y
-CONFIG_SPL_HASH_SUPPORT=y
+CONFIG_SPL_SHA256=y
```

Comment:
Do we need the same for imx6ul mfgtools?

Reply:
@angolini this set is for 6ull only. 6ul support is coming with another patchset.